### PR TITLE
chore: Bundle JavaScript modules

### PR DESCRIPTION
**Pull request by narcis-radu**
The PR is a follow-up on the [recent discussion](https://github.com/adobe/fedpub/pull/8#issuecomment-759470136) we had around code modularization; we want to keep things modular for development purposes and create a code bundle for production.
The code bundle will be automatically created via Github actions after every code commit. The action has to be implemented.

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -3,4 +3,4 @@
 <link id="favicon" rel="icon" type="image/svg+xml" href="/icons/adobe.svg">
 <link rel="stylesheet" href="/hlx_fonts/pnv6nym.css"/>
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<script type="module" src="/scripts.js"></script>
+<script type="module" src="/dist/app.min.js"></script>
```
Comment (overmyheadandbody) [inline head.html:6]: Not sure if this still needs to be a `module`, now that the code in `dist` will be a simple IIFE.

**Review by overmyheadandbody — APPROVED** [review]

**Review by overmyheadandbody — APPROVED** [review]

File: head.html (hunk 1 above)
Comment (rofe) [reply head.html]: I'd still recommend using `defer` or `module` to make sure it's not intefering with the rest of the resoutrces being loaded.

**Comment by rofe** [comment]
These changes look fine in general, but: have you checked the impact on the Lighthouse Score?

**Comment by overmyheadandbody** [comment]
I would say that there shouldn't be any impact to the Lighthouse score. This approach allows us to write everything in a modular way, but the final result is a single uglified JS file. So no additional network calls, no additional framework (everything will be standard ES6), there shouldn't be any major difference to how things were done initially.

File: head.html (hunk 1 above)
Comment (narcis-radu) [reply head.html]: Sounds good, we'll make some tests and add it back.

**Review by rofe — APPROVED** [review]

**Pull request merged**